src/components/Dashboard/TodoList.vue

```diff
@@ -991,6 +991,9 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
 
     if (header) {
       switch (header.name) {
+        case FilingTypes.AGM_LOCATION_CHANGE:
+          await this.loadAgmLocationChange(task)
+          break
```

Comment:
Why do we need this if we can't save a draft?

Reply:
... Unless payment fails, and then can we resume it?

Reply:
^^ This comment isn't blocking this PR or ticket, but it will block the next ticket (18161).

Reply:
This isn't an issue, but I'd like to see what it looks like to confirm that there is a retry payment button (and possibly a cancel payment button, but that will cause another issue).

Do you have a screenshot?

Reply:
I'd like to see a screenshot of a pending AGM Location Change filing in the To Do list -- the action buttons may need to be updated (which we'd do in the next ticket).

Reply:
Yes Sev. Trying to figure out how I can get a pending filing

Reply:
You'll have to simulate a pay error. How about we leave this for the next ticket? Add a specific comment in the next ticket to look at this. Let's move on.

Reply:
I will add comment